crates/wash-cli/src/bin/wash.rs

```diff
@@ -355,3 +442,40 @@ async fn main() {
 fn experimental_error_message(command: &str) -> anyhow::Result<CommandOutput> {
     bail!("The `wash {command}` command is experimental and may change in future releases. Set the `WASH_EXPERIMENTAL` environment variable or `--experimental` flag to `true` to use this command.")
 }
+
+/// Helper for loading plugins. This function does not return an error because it will exit the
+/// process in case of a fatal error. As such, this shouldn't be called after the CLI is parsed.
+async fn load_plugins() -> (SubcommandRunner, PathBuf) {
+    // We need to use env vars here because the plugin loading needs to be initialized before
+    // the CLI is parsed
+    let plugin_dir = match ensure_plugin_dir(std::env::var("WASH_PLUGIN_DIR").ok()).await {
+        Ok(dir) => dir,
+        Err(e) => {
+            eprintln!("Could not load wash plugin directory: {}", e);
```

Comment:
why not `tracing::error`?

Reply:
This one was explicitly because I would expect a CLI to write an error to stderr if it was exiting with 1. But now that I think about it again, so does `tracing::error` so I'll just change it